odxtools/dopbase.py

```diff
@@ -52,11 +53,13 @@ def _resolve_snrefs(self, diag_layer: "DiagLayer") -> None:
     def is_visible(self) -> bool:
         return self.is_visible_raw in (None, True)
 
-    def convert_physical_to_bytes(self, physical_value, encode_state: EncodeState,
+    def convert_physical_to_bytes(self, physical_value: ParameterValue, encode_state: EncodeState,
```

Comment:
convert_physical_to_bytes have a different argument name in each class, physical_value, param_values...

Reply:
yes, and if it lacks proper type annotation it gets confusing rather quickly. (I think there are even instances of functions with the same name doing something slightly different in unrelated classes. This is what I mean with the de-/encoding code being quite messy at the moment.) Anyway, I'd prefer not unifying the parameter names as part of this PR series...